crane/container.go

```diff
@@ -2,13 +2,14 @@ package crane
 
 import (
 	"fmt"
-	"github.com/flynn/go-shlex"
-	"github.com/michaelsauter/crane/print"
 	"io"
 	"os"
 	"path"
 	"strconv"
 	"strings"
+
+	"github.com/flynn/go-shlex"
+	"github.com/michaelsauter/crane/print"
```

Comment:
I'm using goimports, so maybe that's why. It's always separated the remote packages. I can change this if you prefer it.